marquez/airflow.py

```diff
@@ -96,8 +96,9 @@ def compute_endtime(self, execution_date):
         return end_time
 
     def report_jobrun_change(self, dagrun, **kwargs):
+        session = kwargs.get('session')
```

Comment:
pretty minor, maybe define a constant for the string "session"?

Reply:
I prefer to have it there for now, because it's just used in this method, and I do think we are going to get rid of this mechanism (airflow<->marquez jobrun id mapping) soon.